source/api/utilities/blob.md

```diff
@@ -51,7 +51,7 @@ cy.get("input[type=file]").then(($input) => {
 
 ## Getting dataUrl string
 
-**create an `img` element and set its `src` to the `dataUrl`**
+**Create an `img` element and set its `src` to the `dataUrl`**
```

Comment:
Not a typo: just capitalizing the first word in the sentence for consistency with other examples in the docs.